assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/main/AntennaComplianceToolOptions.java

```diff
@@ -55,20 +75,26 @@
      */
     private final String complianceMode;
 
+    /**
+     * The parameters given in the command line.
+     */
+    private final Set<String> parameters;
+
     /**
      * Creates a new instance of {@code AntennaComplianceToolOptions} with the properties
      * provided.
-     *
-     * @param propertiesFilePath the path to the Antenna config file
+     *  @param propertiesFilePath the path to the Antenna config file
      * @param complianceMode     the mode the compliance tool will get executed with
+     * @param parameters
      * @param debugLog           flag whether debug log should be active
      * @param showHelp           flag whether the help message should be printed
      * @param valid              flag whether the command line is valid
      */
-    AntennaComplianceToolOptions(String propertiesFilePath, String complianceMode, boolean debugLog, boolean showHelp, boolean valid) {
+    AntennaComplianceToolOptions(String propertiesFilePath, String complianceMode, Set<String> parameters, boolean debugLog, boolean showHelp, boolean valid) {
         super(debugLog, showHelp, valid);
         this.propertiesFilePath = propertiesFilePath;
         this.complianceMode = complianceMode;
+        this.parameters = parameters;
```

Comment:
To make sure that the AntennaComplianceToolOptions class remains immutable, the set with parameters should not be directly exposed to clients, so that it cannot be manipulated after an instance was created. So create a defensive copy here and in _getParameters()_, e.g. 
`this.parameters = new HashSet(parameters);`
Alternatively, the set can be stored as an unmodifiable set in the field, then it can be returned by _getParameters()_ directly, e.g.
`this.parameters = Collections.unmodifiableSet(new HashSet(parameters));`